go/websocketbridge/websocket_bridge_test.go

```diff
@@ -46,13 +46,15 @@ func FeatureContext(s *godog.Suite) {
 	var clientWebsocket *websocket.Conn
 	var clientCount int
 	var outgoingMessageId string
+	var outgoingActivityId string
 	var savedError error
 
 	s.BeforeScenario(func(interface{}) {
 		exocomPort = freeport.GetPort()
 		clientPort = freeport.GetPort()
 		clientURL = fmt.Sprintf("ws://localhost:%v", clientPort)
 		outgoingMessageId = ""
+		outgoingActivityId = ""
```

Comment:
We can't capture the value of the activity id since it's assigned in exorelay, but this can be removed.